service/src/main/java/bio/terra/workspace/service/resource/controlled/cloud/gcp/dataproccluster/startup.sh

```diff
@@ -731,8 +731,11 @@ if [[ "${SOFTWARE_FRAMEWORK}" == "HAIL" ]]; then
   emit "Installing Hail..."
 
   # Create the Hail install script. The script is based off of Hail's init_notebook.py
-  # script that is executed by 'hailctl dataproc start'. This modified script excludes
-  # the step of starting a jupyter service.
+  # script that is executed by 'hailctl dataproc start'. This modified script omits
+  # the following steps:
+  # - Configuring and starting a custom jupyter systemd service
+  # - Configuring and enabling hail's spark monitor nbextension.
+  # TODO: BENCH-1094: Consider whether we need spark monitor
   cat << EOF >"${HAIL_SCRIPT_PATH}"
 #!${RUN_PYTHON}
 # This modified Hail installation script installs the necessary Hail packages and jupyter extensions,
```

Comment:
> Please add some indication of "why".

Updated PR description. I added a comment on omitting spark monitor, but I left it outside of the user facing hail install script. Perhaps I could add something like: `# Also skip installing spark monitor due to version incompatibility with the latest version of jupyter`?